chef/cookbooks/hyperv/recipes/do_ceilometer.rb

```diff
@@ -0,0 +1,8 @@
+raise if not node[:platform] == 'windows'
+
+include_recipe "hyperv::openstack_install_ceilometer"
+include_recipe "hyperv::openstack_install_neutron"
+include_recipe "hyperv::config_neutron"
+include_recipe "hyperv::config_ceilometer"
+include_recipe "hyperv::register_services_ceilometer"
+include_recipe "hyperv::register_services_neutron"
```

Comment:
Technically makes sense to install Ceilometer agent w/out Nova.  AFAIK Ceilometer can read from Nova, Cinder, Swift, Neutron ... and others.  But I agree that the normal use-case is Nova here.